gc/base/standard/Scavenger.cpp

```diff
@@ -2145,7 +2162,7 @@ MM_Scavenger::completeScan(MM_EnvironmentStandard *env)
 		}
 	}
 
-	env->_scavengerStats.resetCopyScanCounts();
+	Assert_MM_true(env->_scavengerStats._slotsScanned == 0 && env->_scavengerStats._slotsCopied == 0);
```

Comment:
if you have problems asserting during aborted scavenges, than just relax the assert with ' &&  !abort'

Reply:
we could remove this assert anyway, and keep only the after the loop